tutorials/fundamentals/parse-url.md

```diff
@@ -0,0 +1,15 @@
+To parse a URL in JavaScript, use the `new URL()` constructor.
+The constructor will convert parts of the URL into accessible properties.
```

Comment:
"convert parts of the URL into accessible properties" -> "parse the URL into easy-to-use properties"